### PR TITLE
Aligning the old docs with hubflow

### PR DESCRIPTION
**Pull request by bruskiza**

### CODE REVIEW

**Comment by bruskiza** [comment]
@smn @hodgestar @hodgestar it's not perfect, but it ties up the old docs to the cheatsheet.

File: docs/tools.rst [hunk 1]
```diff
@@ -57,6 +57,15 @@ when the feature branch lands, so you will need to update this yourself.
 Please read `Useful Github Patterns <http://blog.quickpeople.co.uk/2013/07/10/useful-github-patterns/>`_
 to see ways of working with branches and pull requests that we like.
 
+Hubflow
+-------
+
+Hubflow is an adapted version of Gitflow, specifically tailored for use with Github.
```
Comment (hodgestar) [inline docs/tools.rst:63]: Nitpick -- `HubFlow`, `GitFlow` and `GitHub` should probably all have capitals in the middle when used as proper nouns.

File: docs/tools.rst [hunk 2]
```diff
@@ -65,6 +74,16 @@ with GitHub_'s API. It allows one to turn issues on GitHub into
 pull-requests. If that is done then once the pull-request is merged into
 the main branch the issue is automatically closed.
 
+
+We use the `Git Flow`_ branching model as part of our development.
```
Comment (hodgestar) [inline docs/tools.rst]: Just checking that the `Git Flow` link is already defined somewhere?

File: docs/tools.rst [hunk 3]
```diff
@@ -57,6 +57,15 @@ when the feature branch lands, so you will need to update this yourself.
...
 to see ways of working with branches and pull requests that we like.
 
+Hubflow
+-------
+
+Hubflow is an adapted version of Gitflow, specifically tailored for use with Github.
+
+It provides the usefulness of `Git Flow`_ with Github goodness embedded.
+
+For more information on that, see this link: `Hubflow <https://datasift.github.io/gitflow/>`
```
Comment (hodgestar) [inline docs/tools.rst]: I think this link is missing an underscore at the end.

File: docs/tools.rst [hunk 4]
```diff
@@ -65,6 +74,16 @@ with GitHub_'s API. It allows one to turn issues on GitHub into
 pull-requests. If that is done then once the pull-request is merged into
 the main branch the issue is automatically closed.
 
+
+We use the `Git Flow`_ branching model as part of our development.
+It's a convenient way to manage your branches. You are not required to use
+Git Flow but you are required to follow naming conventions it sets
+with regard to branch-names and prefixes.
```
Comment (hodgestar) [inline docs/tools.rst]: Branch names shouldn't be hyphenated.

File: docs/tools.rst [hunk 5]
```diff
@@ -65,6 +74,16 @@ with GitHub_'s API. It allows one to turn issues on GitHub into
...
+
+with GitHub_'s API. It allows one to turn issues on GitHub into
```
Comment (hodgestar) [inline docs/tools.rst]: Not sure this paragaph belongs here? The start is unconnected to anything and the rest seems to be about HubFlow maybe?

**Comment by bruskiza** [comment]
@hodgestar re-review?

**Comment by hodgestar** [comment]
@bruskiza In some places we're referencing "git-flow_" and in others "Git Flow_". Which one is correct?

**Comment by bruskiza** [comment]
Thanks @hodgestar sorry... it's a bit of a rats nest.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**